smithy-model/src/main/java/software/amazon/smithy/model/validation/testrunner/SmithyTestSuite.java

```diff
@@ -107,6 +109,15 @@ public static Stream<Object[]> defaultParameterizedTestSource(Class<?> contextCl
                 .parameterizedTestSource();
     }
 
+    public static Result singleFileTestSource(String path, Class<?> contextClass) throws MalformedURLException {
```

Comment:
Docs?

Reply:
Did this method get removed?

Reply:
I wasn't able to comment or resolve it in the prior PR. This method is invaluable for testing and somewhat non-trivial to add every time a dev wants to use it. Maybe it could be marked `@SmithyInternalApi` but there are times I need to add a separate unit test for a specific file in the ErrorsFileTest to avoid other files hitting a break point.